src/web/make-report.rkt

```diff
@@ -125,7 +125,7 @@
             (h2 "Accuracy vs Cost")
             (svg)
             (figcaption "A joint cost-accuracy pareto curve for the "
-                        "Herbie runs below. Accuracy is on the vertical "
+                        "Herbie runs below. Accuracy is on the horizontal "
                         "axis, and cost is on the vertical axis. Down "
```

Comment:
The original caption was definitely nonsensical, but I think this is the opposite of a fix! Accuracy is on the vertical axis & cost is on the horizontal axis.